app/models/piece.rb

```diff
@@ -0,0 +1,7 @@
+class Piece < ActiveRecord::Base
```

Comment:
Should only use it for the STI IMO. Basically you're breaking string type definitions into contextual symbols.

Cool, I'm all for isolated changesets.